src/main/resources/application.properties

```diff
@@ -0,0 +1,6 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/training
+spring.datasource.username= ana_java
+spring.datasource.password= 123
+spring.jpa.properties.hibernate.dialect = org.hibernate.dialect.PostgreSQLDialect
+spring.jpa.hibernate.ddl-auto = update
+server.port = 8081
```

Comment:
Remember to add an EOF in the files that allow it in order to avoid differences in the versioning control. Adding an extra line at the end of the file will get rid of this symbol:  

![image](https://user-images.githubusercontent.com/43755493/74338496-12d09180-4d81-11ea-8db0-6f1a04a67a21.png)

😃

Reply:
:+1: